src/Features/Core/Portable/FindUsages/IRemoteFindUsagesService.cs

```diff
@@ -143,8 +143,8 @@ public SerializableDocumentSpan(DocumentId documentId, TextSpan sourceSpan)
             SourceSpan = sourceSpan;
         }
 
-        public static SerializableDocumentSpan Dehydrate(DocumentSpan documentSpan)
-            => new(documentSpan.Document.Id, documentSpan.SourceSpan);
+        public static SerializableDocumentSpan Dehydrate(DocumentIdSpan documentSpan)
```

Comment:
💡 Can you file a follow-up issue to remove `SerializableDocumentSpan`? I don't want to clutter this pull request, but it should be unnecessary now.